test/Api.Test/Tools/Validators/SendRotationValidatorTests.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Bit.Api.Tools.Models;
+using Bit.Api.Tools.Models.Request;
+using Bit.Api.Tools.Validators;
+using Bit.Core.Entities;
+using Bit.Core.Exceptions;
+using Bit.Core.Tools.Entities;
+using Bit.Core.Tools.Enums;
+using Bit.Core.Tools.Models.Data;
+using Bit.Core.Tools.Repositories;
+using Bit.Core.Tools.Services;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Api.Test.Tools.Validators;
+
+[SutProviderCustomize]
+public class SendRotationValidatorTests
+{
+    [Fact]
+    public async Task ValidateAsync_Success()
+    {
+        // Arrange
+        var sendService = Substitute.For<ISendService>();
+        var sendRepository = Substitute.For<ISendRepository>();
+
+        var sut = new SendRotationValidator(
+            sendService,
+            sendRepository
+        );
+
+        var user = new User { Id = new Guid() };
+        var sends = CreateSends();
+        var userSends = sends.Select(s =>
```

Comment:
⚠️  This line is performing a computation, and then used as part of an assertion on line 42. Use a hard-coded expected result instead.